client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
+		const multipleIsEnabled = Number( multiple ) !== 0;
```

Comment:
@jsnajdr great points. I took out the use of `Number` and left a TODO note to dig into the bigger issue. I'd like to merge this and follow up with that in another PR